test/integration/bugs_test.go

```diff
@@ -20,6 +20,18 @@ const knownFileSuffixesInsideArchiveRegex string = `(`+
 	`(\/|^)(config|id|invoker|metrics|version)` +
 `)$`
 
+//https://bugzilla.redhat.com/show_bug.cgi?id=1841057
+func TestUploadNotDelayedAfterStart(t *testing.T) {
+	time1:=logLineTime(t, `Reporting status periodically to .* every`)
```

Comment:
Nice, Could we please just for sure check that IO operator is healthy at the beginning of TC, because that is precondition for fast upload mode.
Also, at the beginning of Log file should be `It is safe to use fast upload` and not `Not safe for fast upload`, but I guess checking health of operator should be good enough fot test. (To not to have investigate ccases when AWS or OCP itself failed).